contribs/gmf/examples/search.js

```diff
@@ -74,6 +71,8 @@ function MainController(gmfThemes, ngeoFeatureOverlayMgr, ngeoNotification) {
    */
   this.inputValue = '';
 
+  this.ngeoNotification = ngeoMessageNotification;
```

Comment:
Why this you can use `ngeoMessageNotification` in all the file, not?

Reply:
Yes, I need to fix that everywhere we use the new services.